### PR TITLE
feat: adding caching and postgres support

### PR DESCRIPTION
**Pull request by John2360**

### CODE REVIEW

File: examples/test.py [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
+    org_id="300fe35d-e841-4c1d-8956-a7aa5d17ecad",
+    api_key="MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQg45BBe3K2squQ6Nlxjjs3lrLFpMAUkgWY452ej1dOn+CgCgYIKoZIzj0DAQehRANCAARJ/M6H92xhPMpC12KCvdwl8mdL8c94/B1AiJoqzdFn78vZliCOjPWC7Ostxulj4WziweHuTJVL6vE6L4T5EPDN",
 )
 
-response = fortress.create_database("sdk_test")
-if response.success:
-    print("Database created")
-else:
-    print(f"Database creation failed: {response.message}")
+# # Create a database
```
Comment (willothy) [inline examples/test.py]: Should we clean this up or uncomment?

File: fortress_sdk_python/client.py [hunk 2]
```diff
@@ -1,42 +1,336 @@
...
+
+        endpoint = f"{self.base_url}/v1/organization/{self.org_id}/{type}/{id}/uri"
+        response = requests.get(
+            endpoint,
+            headers={"Api-Key": self.api_key},
+        )
+
+        json_response = response.json()
+        if response.status_code != 200:
+            return DatabaseUriResponse(
```
Comment (willothy) [inline fortress_sdk_python/client.py]: Do we want to throw errors?

File: examples/test.py (hunk 1 above)
Comment (John2360) [reply examples/test.py]: Oh dang i did then tested again. Will do

**Comment by willothy** [comment]
> [SDK Python Connection Cacheing](https://www.notion.so/SDK-Python-Connection-Cacheing-607c5e869c7a49b3abc81077e7d11d9c?pvs=8&n=github_linkback)

hmm these are public... not ideal

**Comment by John2360** [comment]
Oh jeez yeah

**Review by willothy — APPROVED** [review]

**Review by willothy — APPROVED** [review]

**Pull request merged**